roles/forklift_versions/molecule/default/molecule.yml

```diff
@@ -0,0 +1,13 @@
+---
+driver:
+  name: delegated
+  options:
+    managed: False
+    ansible_connection_options:
+      ansible_connection: local
+platforms:
+ - name: instance
+#provisioner:
+#  name: ansible
+#verifier:
+#  name: ansible
```

Comment:
Not a fan of commented "code". Is this needed in the future?

Reply:
if it is, it can be added.